is-ios-safari.js

```diff
@@ -2,12 +2,12 @@
 
 var isIos = require('./is-ios');
 
+function isWebkit(ua) {
+  return ua.match(/WebKit/i);
```

Comment:
One optimization we could make is to define this variable outside of this function, so we only create the regex once, and not every time we call `isWebkit`.